tests/components/uvc/test_camera.py

```diff
@@ -225,6 +237,7 @@ def setup_method(self, method):
 
     def test_properties(self):
         """Test the properties."""
+        yield from self.async_added_to_hass()
```

Comment:
You cannot yield inside non-async methods and there is no `async_added_to_hass` method on the test class

Reply:
I have no idea how to test that. You said on Discord that I could test it by setting up the component with `await async_setup_component()` but afaik that doesn't work because `async_setup_component` isn't used in `camera.py`?

I'm really way out of my element in writing tests and am having trouble just comprehending what I'm reading.

So far I've tried the following (basically by copy/pasting from other components) but end up with warnings that the coroutine was never awaited:
```python
    @mock.patch.object(uvc, 'UnifiVideoCamera')
    async def test_async_setup(hass, mock_uvc):
        """Test async setup."""
        result = await async_setup_component(hass, 'camera', {
            'camera': {
                'platform': 'uvc'
            }})
        await hass.async_block_till_done()

        assert result

    async def test_setup_with_no_config(hass):
        """Test that we do not discover anything."""
        assert await async_setup_component(hass, uvc.DOMAIN, {}) is True
        assert uvc.DOMAIN not in hass.data
```

Reply:
```tests/components/uvc/test_camera.py::TestUVCSetup::test_async_setup
  /usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/unittest/case.py:615: RuntimeWarning: coroutine 'TestUVCSetup.test_async_setup' was never awaited
    testMethod()

tests/components/uvc/test_camera.py::TestUVCSetup::test_setup_with_no_config
  /usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/unittest/case.py:615: RuntimeWarning: coroutine 'TestUVCSetup.test_setup_with_no_config' was never awaited
    testMethod()```

Reply:
I don't have time to help you write tests.

Reply:
Well I got this part fixed :shrug: